tethys_gizmos/static/tethys_gizmos/js/tethys_map_view.js

```diff
@@ -1218,25 +1218,30 @@ var TETHYS_MAP_VIEW = (function() {
 
         html += '<li class="legend-class ' + legend_class.type + '">';
         if (legend_class.LEGEND_TYPE === "mvlegendimage") {
-          html += '<span class="legend-class-symbol">' + legend_class.value + '</span>' +
-                  '<span class="legend-class-value"><img src="' + legend_class.image_url + '"></span></li>';
```

Comment:
You left the <li> element started on line 1219 open. In the original code it is closed at the end of line 1222. You don't close it in your new code.